crates/ika-core/src/dwallet_mpc/mpc_session.rs

```diff
@@ -45,6 +45,15 @@ pub(crate) struct ReadyToAdvanceCheckResult {
     pub(crate) malicious_parties: Vec<PartyID>,
 }
 
+/// DWallet MPC session data that is being driven by the event that started this session.
+#[derive(Clone)]
+pub struct EventDrivenData {
```

Comment:
Rename this, MPCEventData is much clearer.
EventDrivenData sounds like a framework.
Fix the comment and the var names.